lib/vquic/msh3.c

```diff
@@ -447,8 +493,13 @@ static CURLcode cf_connect_start(struct Curl_cfilter *cf,
 {
   struct cf_msh3_ctx *ctx = cf->ctx;
   bool insecure = !cf->conn->ssl_config.verifypeer;
+  MSH3_ADDR addr = {0};
+  if(ctx->ai) {
+    memcpy(&addr, ctx->ai->ai_addr, ctx->ai->ai_addrlen);
+  }
+  MSH3_SET_PORT(&addr, (uint16_t)cf->conn->remote_port);
```

Comment:
Is the address info above always specified? Will it already contain the remote port?